projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.spec.ts

```diff
@@ -35,4 +48,10 @@ describe('OpenInAppComponent', () => {
 
     expect(currentLocation).toBe('mockRedirectUrl');
   });
+
+  it('should set the value `mobile_notification_expires_in` in session storage on dialog close', async () => {
+    component.onCloseDialog();
```

Comment:
Can you also check that the local storage was empty before calling a dialog

Reply:
I now have cleared the session storage before each test run to avoid tearing mutated storage